src/main/msp/msp.c

```diff
@@ -758,7 +758,7 @@ static bool mspCommonProcessOutCommand(int16_t cmdMSP, sbuf_t *dst, mspPostProce
             voltageMeterRead(id, &meter);
 
             sbufWriteU8(dst, id);
-            sbufWriteU8(dst, (uint8_t)constrain((meter.displayFiltered + 5) / 10, 0, 255));
+            sbufWriteU16(dst, (uint16_t)constrain((meter.displayFiltered + 5) / 10, 0, 0xFFFF));
```

Comment:
You can't alter the internal structure of an existing MSP message as it will break every client that's not aware of the change - meaning every single client including our own Configurator.

Reply:
So how should I do for this bug?

Reply:
Well first, this is not a bug. It's a limitation of the original design. Second, There is no way to change the `MSP_VOLTAGE_METER` msp message to extend the range because of it's original design. The message is organized so that each meter is expected to be 2 bytes (ID and voltage). So the consumer indirectly determines the number of meters based on the total message size divided by 2. There is no way to change the contents and have existing consumers not fail. So the only solution is to deprecate this message (but still support it) and add a new message that's better designed and adds support for higher voltage and possibly additional data while providing a more extensible design. The problem with this is that no existing client will understand or utilize this new message. So while we can enhance the Configurator to use it, other consumers like the Speedybee app, DJI OSD, etc. will not know about or utilize this message.

Reply:
And clearly this is a lot of work to be able to support the still very limited use case of displaying voltages > 6S in the Configurator.

Reply:
@etracer65 So could I add a new MSP info such as MSP_VOLTAGE_METERS_HIGH and support it?

Reply:
@Linjieqiang: We _could_. The real question here is _do we want to?_ As @etracer65 says, the use case is very much an edge case, so I am not sure if the effort to add and then maintain this is justified. What use case do you have in mind where getting the main battery voltage through `MSP_BATTERY_STATE` (or `MSP_ANALOG`) isn't enough? Both of these support voltages of up to 320V.

Reply:
This is the case, we plan to produce 8S and 12S BLHeli_32 ESCs. We believe that the FPV drones will be used soon, but currently the api protocol only supports ESC voltage of 6s, which is easy to cause misunderstanding by users, so hope to get improvement for it. @mikeller

Reply:
@Linjieqiang: What is the use case for these ESCs that will require `MSP_VOLTAGE_METER` to be changed?